code/game/objects/items/weaponry.dm

```diff
@@ -337,6 +337,32 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 	user.visible_message("<span class='suicide'>[user] is slitting [user.p_their()] own throat with [src]! It looks like [user.p_theyre()] trying to commit suicide!</span>")
 	return (BRUTELOSS)
 
+/obj/item/switchblade/makeshift
+	icon_state = "switchblade_ms"
+	desc = "A concealable spring-loaded knife."
+	force = 2
+	throwforce = 3
+	
+/obj/item/switchblade/makeshift/attack_self(mob/user)
+	extended = !extended
+	playsound(src.loc, 'sound/weapons/batonextend.ogg', 50, 1)
+	if(extended)
+		force = 15
+		w_class = WEIGHT_CLASS_NORMAL
+		throwforce = 15
+		icon_state = "switchblade_ext_ms"
+		attack_verb = list("slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut")
+		hitsound = 'sound/weapons/bladeslice.ogg'
+		sharpness = IS_SHARP
+	else
+		force = 1
```

Comment:
Now, you'll have to change the equivalent line on `switchblade/attack_self` to 
```
		force = initial(force)
```
lest the retracted makeshift switchblade force reset to the base non-makeshift switchblade instead.